### ISSUE TITLE
Installers are from unknown/scary distributors

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt#2087]
**Description of the desired feature**

Both the Windows and Mac installers runs into the system police.  For Windows you are told the installer may damage your system and you have to click the "More info" to even see the "Run anyway" button to run the installer.  On macOS you are told you cannot run the installer and must first to to the system preferences to relax this or make an install anyway selection.

Is it possible for us to become a trusted vendor? I don't even know what that means so this is a question from ignorance.

**Comment by seisman** [comment]
For macOS, someone has to pay $99/yr to join Apple's Developer Program. See https://discussions.apple.com/thread/7488189 and https://developer.apple.com/support/compare-memberships/.

**Comment by PaulWessel** [comment]
I think I can do that.  It also says if you are educational and you distribute free stuff then the fee can be waived.  I will see if I can have this registered as a generic user instead of my personal account.

Hopefully something similar for Microsoft perhaps, @joa-quim ?

**Comment by joa-quim** [comment]
Don't know. But I never saw the Win installer trigger that policy threat message.

**Comment by PaulWessel** [comment]
Your security settings are probably more relaxed.  During the GMT workshop on installers for Windows today we have a few issues:

For one person, double-clicking the installer did not start it [had to override a setting]
When I did it myself on my Dell Win 10 I had to click the "More Info" text to get the second button that says Run anyway.  The only button that showed on default was do not install.
